packages/view/backend/sirius-components-view-form/src/main/resources/model/form.ecore

```diff
@@ -247,6 +247,19 @@
     <eStructuralFeatures xsi:type="ecore:EReference" name="body" upperBound="-1" eType="ecore:EClass ../../../../../sirius-components-view/src/main/resources/model/view.ecore#//Operation"
         containment="true"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="DateTimeDescription" eSuperTypes="#//WidgetDescription">
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="stringValueExpression"
```

Comment:
Why did you call that reference `stringValueExpression`? shouldn't it be just `valueExpression`?

Reply:
Because it is a date, so I wanted to distinguish a date in format string from a date in other format.

Reply:
Ok